contracts/MarginCalculator.sol

```diff
@@ -21,18 +20,40 @@ contract MarginCalculator {
     using SafeMath for uint256;
     using FPI for FPI.FixedPointInt;
 
-    address public addressBook;
+    /// @dev oracle module
+    OracleInterface public oracle;
```

Comment:
does this mean if we change the oracle we have to redeploy margin calculator?

Reply:
yeah